### DISCUSSION TITLE
关于transport layer的使用疑问

### DISCUSSION THREAD

**Discussion by 359547629** [alibaba/xquic#227] [Q&A] [answered]
我想使用xquic的transport layer来实现服务端/客户端 网络通讯：
1. 出于性能考虑， 是否有选项可关闭SSL？ 比如：传输数据 不加密
2. 若客户端向同一个服务端发起多个连接(局域网)， 每个连接均需发送大量数据，是应该：
    a. 创建多个connection，每个connection创建一个stream？
    b. 还是只需一个connection, 然后创建多个stream?
  哪个传输性能更高?

期望回复，谢谢！

**Comment by ruiqizhou** [comment, marked as answer]
`-N` 参数可实现 0-RTT 和 1-RTT 不加密，握手过程加密，参见 https://github.com/alibaba/xquic/blob/main/docs/docs-zh/Testing-zh.md

**Comment by parthibx24** [comment]
@359547629 Hello, were you able to implement only the quic transport to build server/client? I am kind of lost looking at the test codes. Some code snippets for server/client would be really helpful!
